docs/index.mdx

```diff
@@ -39,6 +39,10 @@ import Box from 'superbox'
 <button>code example</button>
 ```
 
+```ruby
+puts "This is a Ruby Example"
+```
```

Comment:
I think I'll probably do a complete pass through this demo doc and change things quite a bit, so feel free to use it for development, but don't worry about having examples of all the features in here for the time being